rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml

```diff
@@ -70,3 +70,21 @@
     msg: "One or more of the cinder services are in the down state. Check the output above for more detail."
   when: "'down' in '{{ item.split(' ')[-1] }}'"
   with_items: "{{ cinder_output.stdout_lines | default([]) }}"
+
+# Adds orchestration service checks per https://rpc-openstack.atlassian.net/browse/UG-543
+- name: Grab relevant output of orchestration service list
+  shell: |
+    . ~/openrc
+    openstack orchestration service list --insecure
```

Comment:
I think requesting a JSON response provides a better experience because we can reference the specific key instead of searching the text, e.g.:

```yaml
    openstack orchestration service list --insecure  -f json
```